src/_data/catalog/widgets.json

```diff
@@ -171,7 +171,7 @@
   },
   {
     "name": "App bar",
-    "description": "Top app bars display information and actions at the top of a screen.",
+    "description": "Container that displays content and actions at the top of a screen.",
```

Comment:
**question:** Is this persistent? If it doesn't change throughout the app's use, then add "Persisent" to the beginning of this line. Otherwise, LGTM.